src/patternfly/utilities/Text/examples/Text.md

```diff
@@ -123,19 +229,18 @@ Text utility
 | `.pf-u-color-200`           | `*`        | Sets font-color to Color 200           |
 | `.pf-u-color-300`           | `*`        | Sets font-color to Color 300           |
 | `.pf-u-color-400`           | `*`        | Sets font-color to Color 400           |
-| `.pf-u-icon-color-light`    | `*`        | Sets font-color to Icon Color Light    |
-| `.pf-u-icon-color-dark`     | `*`        | Sets font-color to Icon Color Dark     |
-| `.pf-u-link-color`          | `*`        | Sets font-color to Link Color          |
 | `.pf-u-active-color-100`    | `*`        | Sets font-color to Active Color 100    |
-| `.pf-u-active-color-200`    | `*`        | Sets font-color to Active Color 200    |
-| `.pf-u-active-color-300`    | `*`        | Sets font-color to Active Color 300    |
```

Comment:
Are we removing `active-color-200/300`? If so, looks like they are still in the CSS:

https://github.com/patternfly/patternfly/blob/e3962b3df7ada7a975af955dca7ac07146169eff/src/patternfly/utilities/Text/text.scss#L98-L103